### PR TITLE
Welcome max bot

### PR DESCRIPTION
**Pull request by wsf235**
syntax: !loop [time in seconds] [message to be looped]

example: !loop 20 Max-bot is constantly in operation and will loop this statement every 20 seconds

I feel as though someone can come along later and upgrade this function to maybe include more loops or format it better for time.

### CODE REVIEW

File: bot/client.js [hunk 1]
```diff
@@ -60,6 +62,42 @@ client.on('message', (message) => {
...
+    let result = newMessage.content.substr(newMessage.content.indexOf(" ") + 1);
+    let result2 = result.substr(result.indexOf(" ") + 1);
+    result = result.split(" ");
+    result = Number(result[0]);
+    util.log(result);
+    util.log(result2);
+
+    // Check if the pre-existing toggle is true or false
+    // If the toggle is true, then set the toggle to false and turn off the loop
+    if (toggle !== false) {
```
Comment (Codemax999) [inline bot/client.js]: Can simplify with `if (toggle) ...`

File: bot/client.js [hunk 2]
```diff
@@ -60,6 +62,42 @@ client.on('message', (message) => {
...
+      clearInterval(interval);
+      newMessage.channel.send("Loop is off");
+      toggle = false;
+    } else if (toggle !== true) {
```
Comment (Codemax999) [inline bot/client.js]: Can simplify with `else if (!toggle)...`

**Review by Codemax999 — COMMENTED** [review]
Awesome, added some small things about handling booleans. Other than that LGTM!

**Comment by wsf235** [comment]
Thank you much!
And for the most part, the way I handled the booleans was more so I could keep track of what I was doing in that case. Using toggle and !toggle sometimes mixes me up.

**Pull request merged**